pkg/pod/admission/limit_request.go

```diff
@@ -0,0 +1,98 @@
+package admission
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"k8s.io/kubernetes/pkg/admission"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/resource"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+
+	"github.com/ghodss/yaml"
+	"speter.net/go/exp/math/dec/inf"
+)
+
+func init() {
+	admission.RegisterPlugin("PodLimitRequest", func(c kclient.Interface, config io.Reader) (admission.Interface, error) {
+		return newPodLimitRequest(config)
+	})
+}
+
+type limitRequestConfig struct {
+	Enabled                   bool    `json:"enabled"`
+	LimitCpuToMemoryRatio     float64 `json:"limitCpuToMemoryRatio"`
+	CpuRequestToLimitRatio    float64 `json:"cpuRequestToLimitRatio"`
+	MemoryRequestToLimitRatio float64 `json:"memoryRequestToLimitRatio"`
+}
+type podLimitRequest struct {
+	*admission.Handler
+	limitRequestConfig
```

Comment:
Give this an explicit name.  It will make a future transition to config cleaner.